readme.md

```diff
@@ -57,17 +57,18 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Hello_Microverse] <a name="about-project"></a>
+# 📖 [Hello Microverse Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] As suggested by the previous reviewer in your other pull request, kindly remove all the default comments from your readme as you have tried to do because they were used in the template to explain each section's purpose to the user. You can go ahead, check through all the sections and remove the default comments from your README

e.g 
![image](https://user-images.githubusercontent.com/63473584/226713433-72a8e0e5-5c39-4bc3-a765-8e4379c5b253.png)